src/e3/os/process.py

```diff
@@ -601,7 +601,7 @@ def is_running(self) -> bool:
         else:
             return self.internal.is_running()
 
-    def children(self) -> List[psutil.Process]:
+    def children(self) -> List[Any]:
```

Comment:
I wish we had a way to get a reminder to check this again once psutil.Process gets type hinting... But I can't think of any way other than having a TODO item to periodically do a review.

Could we perhaps declare a `TypeHintingMissing` type equal to Any, and use that instead? That way, at least, it would be easy to grep for them, and find all the locations where we used `Any` because type hinting was missing. What do people think?

Reply:
I doubt psutil will have type annotation in the near future